index.js

```diff
@@ -6,14 +6,13 @@ var AWS = require("aws-sdk"),
 	fs = require("fs"),
 	request = require("request"),
 	path = require("path"),
+	sharp = module.parent.require("sharp"),
```

Comment:
NPM will sometimes install it in the nodebb -> node_modules folder but it should always put a reference to it in the s3-uploader -> node_modules -> bin folder as far as I'm aware. This sounds more like a bug with heroku and npm than the code. I'm also reticent to try and add fixes for specific platforms as the plugin needs to support all platforms.